addons/website_form/static/src/js/website_form_editor.js

```diff
@@ -166,6 +167,61 @@ odoo.define('website_form_editor', function (require) {
             });
         },
 
+        /**
+         * Prompt the user for a default value for this field.
+         *
+         * It creates a modal with a clone of the target field.
+         */
+        ask_default_value: function () {
+            this.trigger_up("clean_for_save");
+            var $body = $("<div/>").append(this.$target.clone());
+            // Alter unique ids, for datepickers
+            $body.find("[id]").each(function () {
```

Comment:
Why reassigning new ids each time we ask for a default value ?

Reply:
Because if the data-target matches a different datetimepicker, you'd be altering the value of the wrong input.